modules/oxide/src/less/theme/content/selection/selection.less

```diff
@@ -69,20 +71,42 @@
   // Selection of whole table cells
   td[data-mce-selected],
   th[data-mce-selected] {
-    background-color: @table-selection-color !important;
+    outline: none; //Prevent selection to be visible
+    position: relative;
+    user-select: none; //Prevent selection to be visible
```

Comment:
This is quite likely going to break things, as the content needs to be selectable. I suspect this is why some tests are failing.

Reply:
Removed